lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -4,34 +4,36 @@ hasClass = (node, search) ->
 nodeIsDisabled = (node) ->
    node.getAttribute('disabled') || hasClass(node, 'disabled')
 
-setupRemoteFromTarget = (target, httpRequestType, urlAttribute, form = null) ->
-  httpUrl = target.getAttribute(urlAttribute)
+setupRemoteFromTarget = (target, httpRequestType, form = null) ->
+  httpUrl = target.getAttribute('href') || target.getAttribute('action')
 
-  throw new Error("Turbograft developer error: You did not provide a URL ('#{urlAttribute}' attribute) for tg-remote") unless httpUrl
+  throw new Error("Turbograft developer error: You did not provide a URL ('data-tg-#{urlAttribute}' attribute) for data-tg-remote") unless httpUrl
 
-  if target.getAttribute("remote-once")
+  if target.getAttribute("data-tg-remote-once") || target.getAttribute("remote-once")
```

Comment:
Though because of attributes like `tg-remote` this wouldn't quite work.

Maybe something like
```coffeescript
getTGAttribute(node, attr): ->
  tgAttr = if attr.test[0...3] == 'tg-'
    "data-#{attr}"
  else
    "data-tg-#{attr}"

  node.getAttribute(tgAttr) || node.getAttribute(attr)
```